### DISCUSSION TITLE
Where can I find the source code of remark-mdc?

### DISCUSSION THREAD

**Discussion by Boudewijn26** [nuxt/content#1819] [Q&A]
I see nuxt content is using [remark-mdc](https://www.npmjs.com/package/remark-mdc), but the [link on npm sends me to a 404](https://github.com/nuxtlabs/remark-mdc). Looking at the package.json of remark-mdc I do see it licensed as MIT

The reason I'm asking is because I think I found a bug with component slots with lists in them. In short we do something like:
```markdown
::vue-component
#slot1

* List item 1
* List item 2

#slot2

* Different list item 1
* Different list item 2
::
```
```vue
<template>
  <div>
    <ContentSlot :use="$slots.slot1" />
    <ContentSlot :use="$slots.slot2" />
  </div>
</template>
```
Which I'd expect to result in

```html
<div>
  <ul>
    <li>List item 1</li>
    <li>List item 2</li>
  </ul>
  <ul>
    <li>Different list item 1</li>
    <li>Different list item 2</li>
  </ul>
</div>
```

But instead results in
```html
<div>
  <ul>
    <li>List item 1</li>
    <li>List item 2</li>
  </ul>
  <li>Different list item 1</li>
  <li>Different list item 2</li>
</div>
```

It would appear to me that the markdown ignores the `#slot2` and continues the list. Which I think would be a bug in remark-mdc, but without access to the source code it is difficult to verify. Does anyone know how we could get access to it?

**Comment by abdonrd** [comment]
Are there plans to make the repository public? Thank you!

https://github.com/nuxtlabs/remark-mdc
